### ISSUE TITLE
Sending stuck at "Signing Transaction"

### ISSUE THREAD

**Issue by Giszmo** [mycelium-com/wallet-android#654]
When trying to send transaction, I don't get past the "Signing Transaction" screen.

This is with v3.16.0.16 on all BTC HD accounts.

Elvis can't reproduce.

I'm stuck since hours.

Receiving address type does not affect the result.

**Comment by DrrDino** [comment]
Yes that’s how mine was acting up but thanks to blackhathacker111 on telegram who helped me message him https://t.me/Blackhathacker111
